Data/Repository/Repository.cs

```diff
@@ -34,9 +34,9 @@ public Task<bool> AnyAsync(Expression<Func<TEntity, bool>> predicate)
             return DbSet.AnyAsync(predicate);
         }
 
-        public TEntity GetById(int id)
+        public async Task<TEntity> GetById(int id)
         {            
-            return DbSet.Single(x => x.Id == id);
+            return await DbSet.SingleAsync(x => x.Id == id);
```

Comment:
Good point! I forgot about it :)

Reply:
resolve by https://github.com/anowakowski/PictureApp_API/pull/17/commits/a5e9b125df36d84fc4586c585d9f5f7a4b681585

Reply:
the problem was in duplicate services catalog probably after merged master to UserControllerWithFunctionality repository. The changeset: 
https://github.com/anowakowski/PictureApp_API/pull/17/commits/f864a209dd9450ad835e3936c27f957e258312da should be fix the problem